javascript/javascript1/week2/school.js

```diff
@@ -0,0 +1,121 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName === "Queen") {
+        console.log('Welcome Your Highness');
+    }
+
+
+    else if (studentName === " ") {
+        console.log('Please enter your name');
+    }
+
+
+    else if (class07Students.length > 5) {
+        console.log('Cannot add more students to class 07');
+
+    } else if (existingStudent(studentName)) {
+        console.log('We already have ' + studentName + ' in the class')
+    }
+
+
+
+    else {
+        return class07Students.push(studentName);
+    }
+
+}
+addStudentToClass('Slavia')
+addStudentToClass('Mahi')
+addStudentToClass(' ')
+addStudentToClass('Norman')
+addStudentToClass('Ethan')
+addStudentToClass('Mahi')
+addStudentToClass('Nicole')
+addStudentToClass('Stacia')
+
+addStudentToClass('Sean')
+addStudentToClass('Ava')
+addStudentToClass('Queen')
+
+
+console.log(class07Students);
+
+function existingStudent(student) {
+    for (i = 0; i < class07Students.length; i++) {
+        if (class07Students[i] === student) {
+            return true;
+        }
+
```

Comment:
You might consider returning false when there is no students.
```suggestion
return false
```

Reply:
Yes ...I did it now but I was little confused about it before as when I was adding else statements as false it was returning me the opposite value, but returning the function with true gave me the correct answer